src/pages/builder/Builder.jsx

```diff
@@ -1,3 +1,170 @@
+import GjsEditor from '@grapesjs/react';
+import gjsForms from 'grapesjs-plugin-forms';
+import './customStyles.module.css'
+
+const gjsOptions = {
+    height: '100vh',
```

Comment:
let's please add the following property here. It will allow us to [customize the initial blocks styles](https://github.com/GrapesJS/grapesjs/issues/922).

```
canvas: {
      styles: ['/canvas.css']
    },
```